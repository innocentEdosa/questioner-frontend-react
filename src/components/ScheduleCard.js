import React from 'react';
import PropTypes from 'prop-types';

const ScheduleCard = ({
  style,
  gettingRsvp,
  rsvpResponse,
  rsvpMeetup
}) => {
  const onClickHandler = (e) => {
    e.preventDefault();
    return rsvpMeetup(e.target.innerText);
  };
  return (
    <div style={style} className="specificMeetupRsvp">
      <div className="specificMeetupRsvp-back">
        <div className="btn-group" role="group" aria-label="Third group">
          <button
            onClick={onClickHandler}
            type="button"
            className="btn specificMeetupRsvpBtn"
          >
            Yes
          </button>
        </div>
        <div className="btn-group mx-4" role="group" aria-label="Third group">
          <button
            onClick={onClickHandler}
            type="button"
            className="btn specificMeetupRsvpBtn"
          >
            Maybe
          </button>
        </div>
        <div className="btn-group" role="group" aria-label="Third group">
          <button
            onClick={onClickHandler}
            type="button"
            className="btn specificMeetupRsvpBtn"
          >
            No
          </button>
        </div>
      </div>
      <div className="specificMeetupRsvp-side specificMeetupRsvp-front">
        {gettingRsvp ? (
          <span className="spinner-border text-light" role="status" />
        ) : rsvpResponse ? (
          rsvpResponse
        ) : (
          'Add to schedule'
        )}
      </div>
    </div>
  );
};

ScheduleCard.propTypes = {
  style: PropTypes.shape({}).isRequired,
  gettingRsvp: PropTypes.bool.isRequired,
  rsvpResponse: PropTypes.string.isRequired,
  rsvpMeetup: PropTypes.func.isRequired
};

export default ScheduleCard;
