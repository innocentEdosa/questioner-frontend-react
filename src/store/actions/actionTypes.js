export const LOADING = 'LOADING';
export const AUTH_SUCCESSFUL = 'AUTH_SUCCESSFUL';
export const AUTH_FAILED = 'AUTH_FAILED';
export const AUTH_NAV = 'AUTH_NAV';
export const OPEN_MEETUP_MODAL = 'OPEN_MEETUP_MODAL';
export const CLOSE_MEETUP_MODAL = 'CLOSE_MEETUP_MODAL';
export const CREATE_MEETUP_START = 'CREATE_MEETUP_START';
export const CREATE_MEETUP_SUCCEEDED = 'CREATE_MEETUP_SUCCEEDED';
export const CREATE_MEETUP_FAILED = 'CREATE_MEETUP_FAILED';
export const FETCH_MEETUPS_START = 'FETCH_MEETUPS_START';
export const FETCH_MEETUPS_SUCCEEDED = 'FETCH_MEETUPS_SUCCEEDED';
export const FETCH_MEETUPS_FAILED = 'FETCH_MEETUPS_FAILED';
export const FETCH_RANDOM_MEETUPS = 'FETCH_RANDOM_MEETUPS';
export const FETCH_RANDOM_MEETUPS_START = 'FETCH_RANDOM_MEETUPS_START';
export const FETCH_RANDOM_MEETUPS_SUCCEEDED = 'FETCH_RANDOM_MEETUPS_SUCCEEDED';
export const FETCH_RANDOM_MEETUPS_FAILED = 'FETCH_RANDOM_MEETUPS_FAILED';
export const FETCH_TRENDING_MEETUPS = 'FETCH_TRENDING_MEETUPS';
export const FETCH_TRENDING_MEETUPS_START = 'FETCH_TRENDING_MEETUPS_START';
export const FETCH_TRENDING_MEETUPS_SUCCEEDED = 'FETCH_TRENDING_MEETUPS_SUCCEEDED';
export const FETCH_TRENDING_MEETUPS_FAILED = 'FETCH_TRENDING_MEETUPS_FAILED';
export const FETCH_SPECIFIC_MEETUP_START = 'FETCH_SPECIFIC_MEETUP_START';
export const FETCH_SPECIFIC_MEETUP_SUCCEEDED = 'FETCH_SPECIFIC_MEETUP_SUCCEEDED';
export const FETCH_SPECIFIC_MEETUP_FAILED = 'FETCH_SPECIFIC_MEETUP_FAILED';
export const CREATE_QUESTION_START = 'CREATE_QUESTION_START';
export const CREATE_QUESTION_SUCCEEDED = 'CREATE_QUESTION_SUCCEEDED';
export const CREATE_QUESTION_FAILED = 'CREATE_QUESTION_FAILED';
export const FETCH_MEETUP_QUESTIONS_START = 'FETCH_MEETUP_QUESTIONS';
export const FETCH_MEETUP_QUESTIONS_SUCCEEDED = 'FETCH_MEETUP_QUESTIONS_SUCCEEDED';
export const FETCH_MEETUP_QUESTIONS_FAILED = 'FETCH_MEETUP_QUESTIONS_FAILED';
export const FETCH_ADMIN_MEETUPS_START = 'FETCH_ADMIN_MEETUPS_START';
export const FETCH_ADMIN_MEETUPS_SUCCEEDED = 'FETCH_ADMIN_MEETUPS_SUCCEEDED';
export const FETCH_ADMIN_MEETUPS_FAILED = 'FETCH_ADMIN_MEETUPS_FAILED';
export const VERIFY_USER_SUCCEEDED = 'VERIFY_USER_SUCCEEDED';
export const VERIFY_USER_START = 'VERIFY_USER_START';
export const VERIFY_USER_FAILED = 'VERIFY_USER_FAILED';
export const LOG_USER_OUT = 'LOG_USER_OUT';
export const UPVOTE_QUESTION_START = 'UPVOTE_QUESTION_START';
export const UPVOTE_QUESTION_SUCCEEDED = 'UPVOTE_QUESTION_SUCCEEDED';
export const UPVOTE_QUESTION_FAILED = 'UPVOTE_QUESTION_FAILED';
export const FETCH_MEETUP_RSVP_START = 'FETCH_MEETUP_RSVP_START';
export const FETCH_MEETUP_RSVP_SUCCEEDED = 'FETCH_MEETUP_RSVP_SUCCEEDED';
export const FETCH_MEETUP_RSVP_FAILED = 'FETCH_MEETUP_RSVP_FAILED';
export const CREATE_RSVP_START = 'CREATE_RSVP_START';
export const CREATE_RSVP_SUCCESSFUL = 'CREATE_RSVP_SUCCESSFUL';
export const CREATE_RSVP_FAILED = 'CREATE_RSVP_FAILED';
export const FETCH_USER_RSVP_START = 'FETCH_USER_RSVP_START';
export const FETCH_USER_RSVP_SUCCEEDED = 'FETCH_USER_RSVP_SUCCEEDED';
export const FETCH_USER_RSVP_FAILED = 'FETCH_USER_RSVP_FAILED';
