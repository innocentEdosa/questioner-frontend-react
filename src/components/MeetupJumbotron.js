import React from 'react';
import PropTypes from 'prop-types';
import ScheduleCard from './ScheduleCard';
import { dateFormatter } from '../helper/formatMeetup';

const MeetupJumbotron = ({ meetup }) => (
  <div className="jumbotron jumbotron-fluid specificMeetup">
    <div className="container">
      <h1 className="mt-md-n2 specificMeetupMainText">{meetup.topic}</h1>
      <p className="lead specificMeetupSubText pt-1 pb-3">
        {meetup.happeningOn
          ? `happening on ${dateFormatter(meetup.happeningOn)}`
          : null}
      </p>
      <ScheduleCard style={{ width: '39%' }} />
    </div>
  </div>
);

MeetupJumbotron.propTypes = {
  meetup: PropTypes.shape({}).isRequired
};

export default MeetupJumbotron;
