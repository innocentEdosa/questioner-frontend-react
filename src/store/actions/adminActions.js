import React from 'react';
import { Redirect } from 'react-router-dom';
import { toast } from 'react-toastify';
import * as actions from './actionTypes';
import { postMeetup } from '../api/index';


export const openModal = () => ({
  type: actions.OPEN_MEETUP_MODAL,
  shouldModalOpen: true
});

export const openMeetupModal = () => (dispatch) => {
  dispatch(openModal());
  return <Redirect to="/Admin" />;
};

export const closeMeetupModal = () => ({
  type: actions.CLOSE_MEETUP_MODAL,
  shouldModalOpen: false
});

export const createMeetupStart = () => ({ type: actions.CREATE_MEETUP_START });

export const createMeetupSucceeded = meetup => ({
  type: actions.CREATE_MEETUP_SUCCEEDED,
  meetup,
});

export const createMeetupFailed = error => ({
  type: actions.CREATE_MEETUP_FAILED,
  error
});
export const createMeetup = formInput => async (dispatch) => {
  dispatch(createMeetupStart());
  const {
    title, date, location, description, file
  } = formInput;
  const image = file[0];
  const formData = new FormData();
  formData.append('topic', title);
  formData.append('happeningOn', date);
  formData.append('images', image);
  formData.append('location', location);
  formData.append('description', description);
  try {
    const response = await postMeetup(formData);
    if (response) {
      const meetup = response.data.data;
      dispatch(createMeetupSucceeded(meetup));
      dispatch(closeMeetupModal());
      toast.success('Your meetup was created successfully');
    }
  } catch (err) {
    const error = err.response.data;
    dispatch(createMeetupFailed(error));
  }
};
