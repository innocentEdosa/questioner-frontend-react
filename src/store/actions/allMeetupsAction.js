import { toast } from 'react-toastify';
import * as actions from './actionTypes';
import { getMeetups } from '../api/index';

export const getAllMeetupsStart = () => ({
  type: actions.FETCH_MEETUPS_START
});

export const getAllMeetupsSucceeded = meetups => ({
  type: actions.FETCH_MEETUPS_SUCCEEDED,
  meetups,
});

export const getAllMeetupsFailed = error => ({
  type: actions.FETCH_MEETUPS_FAILED,
  error
});

export const getAllMeetups = () => async (dispatch) => {
  dispatch(getAllMeetupsStart());
  try {
    const response = await getMeetups();
    if (response) {
      const meetups = response.data.data;
      dispatch(getAllMeetupsSucceeded(meetups));
    }
  } catch (err) {
    const error = err.response.data;
    dispatch(getAllMeetupsFailed(error));
    toast.failed('Something went wrong! Cant get all meetups now');
  }
};
