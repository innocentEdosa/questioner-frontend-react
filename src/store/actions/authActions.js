import React from 'react';
import { Redirect } from 'react-router-dom';
import { toast } from 'react-toastify';
import * as actions from './actionTypes';
import { signup, login } from '../api/index';

export const authStart = () => ({ type: actions.LOADING });

export const authSuccessful = (token, user) => ({
  type: actions.AUTH_SUCCESSFUL,
  token,
  user
});

export const authFailed = error => ({
  type: actions.AUTH_FAILED,
  error
});

export const authType = authform => ({
  type: actions.AUTH_NAV,
  authType: authform
});

export const signUp = (username, email, password) => async (dispatch) => {
  dispatch(authStart());
  const authData = {
    username,
    email,
    password,
    firstname: 'yourfirstname',
    lastname: 'yourlastname',
    othername: 'johndoe',
    phoneNumber: 7000000000
  };
  try {
    const response = await signup(authData);
    if (response) {
      const { token, user } = response.data.data[0];
      localStorage.setItem('user', user);
      localStorage.setItem('token', token);
      dispatch(authSuccessful(token, user));
      toast.success('Your sign up was successful');
    }
  } catch (err) {
    const { error } = err.response.data;
    dispatch(authFailed(error));
  }
};

export const logIn = (email, password) => async (dispatch) => {
  dispatch(authStart());
  const authData = {
    email,
    password,
  };
  try {
    const response = await login(authData);
    if (response) {
      const { token, user } = response.data.data[0];
      localStorage.setItem('user', user);
      localStorage.setItem('token', token);
      dispatch(authSuccessful(token, user));
      toast.success('Your log in was successful');
    }
  } catch (err) {
    const { error } = err.response.data;
    dispatch(authFailed(error));
  }
};

export const authNav = authform => async (dispatch) => {
  dispatch(authType(authform));
  return <Redirect to="/Auth" />;
};
