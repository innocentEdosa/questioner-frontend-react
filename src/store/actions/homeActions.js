import * as actions from './actionTypes';
import { fetchRandomMeetups, fetchTrendingMeetups } from '../api/index';

export const getRandomMeetupsStart = () => ({
  type: actions.FETCH_RANDOM_MEETUPS_START
});

export const getRandomMeetupsSucceeded = randomMeetups => ({
  type: actions.FETCH_RANDOM_MEETUPS_SUCCEEDED,
  randomMeetups,
});

export const getTrendingMeetupsStart = () => ({
  type: actions.FETCH_TRENDING_MEETUPS_START
});

export const getTrendingMeetupsSucceeded = trendingMeetups => ({
  type: actions.FETCH_TRENDING_MEETUPS_SUCCEEDED,
  trendingMeetups,
});
export const getRandomMeetups = (number = 4) => async (dispatch) => {
  dispatch(getRandomMeetupsStart());
  try {
    const response = await fetchRandomMeetups(number);
    const randomMeetups = response.data.data;
    dispatch(getRandomMeetupsSucceeded(randomMeetups));
  } catch (error) {
    // console.log(error.response.data);
  }
};

export const getTrendingMeetups = () => async (dispatch) => {
  dispatch(getTrendingMeetupsStart());
  try {
    const response = await fetchTrendingMeetups();
    if (response) {
      const trendingMeetups = response.data.data;
      dispatch(getTrendingMeetupsSucceeded(trendingMeetups));
    }
  } catch (error) {
    // console.log(error);
  }
};
