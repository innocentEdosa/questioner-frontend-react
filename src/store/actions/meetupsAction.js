import { toast } from 'react-toastify';
import * as actions from './actionTypes';
import { getMeetups, fetchSpecificMeetup } from '../api/index';

export const getAllMeetupsStart = () => ({
  type: actions.FETCH_MEETUPS_START
});

export const getAllMeetupsSucceeded = meetups => ({
  type: actions.FETCH_MEETUPS_SUCCEEDED,
  meetups,
});

export const getAllMeetupsFailed = error => ({
  type: actions.FETCH_MEETUPS_FAILED,
  error
});

export const getSpecificMeetupStart = () => ({
  type: actions.FETCH_SPECIFIC_MEETUP_START
});

export const getSpecificMeetupSucceeded = meetup => ({
  type: actions.FETCH_SPECIFIC_MEETUP_SUCCEEDED,
  meetup
});

export const getSpecificMeetupFailed = () => ({
  type: actions.FETCH_SPECIFIC_MEETUP_FAILED
});
export const getAllMeetups = () => async (dispatch) => {
  dispatch(getAllMeetupsStart());
  try {
    const response = await getMeetups();
    if (response) {
      const meetups = response.data.data;
      dispatch(getAllMeetupsSucceeded(meetups));
    }
  } catch (err) {
    const error = err.response.data;
    dispatch(getAllMeetupsFailed(error));
    toast.success('Something went wrong! Cant get all meetups now');
  }
};

export const getSpecificMeetup = id => async (dispatch) => {
  dispatch(getSpecificMeetupStart());
  return fetchSpecificMeetup(id)
    .then((response) => {
      const meetup = response.data.data[0];
      dispatch(getSpecificMeetupSucceeded(meetup));
    })
    .catch((error) => {
      dispatch(getSpecificMeetupFailed());
    });
};
