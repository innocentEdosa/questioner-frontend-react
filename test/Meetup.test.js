import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import MockAdapter from 'axios-mock-adapter';
import thunk from 'redux-thunk';
import axios from 'axios';
// import { shallow } from 'enzyme';
import ScheduleCard from '../src/components/ScheduleCard';
import MeetupJumbotron from '../src/components/MeetupJumbotron';
import * as actions from '../src/store/actions/meetupsAction';
import * as types from '../src/store/actions/actionTypes';
import AllMeetupReducer from '../src/store/reducers/meetupsReducer';

const mockStore = configureStore([thunk]);
const mock = new MockAdapter(axios);
describe('<ScheduleCard />', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<ScheduleCard style={{}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});


describe('<MeetupJumbotron />', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<MeetupJumbotron meetup={{}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});


describe('Meetup actions', () => {
  it('should create an action of type FETCH_MEETUPS_START', () => {
    const expectedAction = {
      type: types.FETCH_MEETUPS_START
    };
    expect(actions.getAllMeetupsStart()).toEqual(expectedAction);
  });

  it('should create an action of type FETCH_MEETUPS_SUCCEEDED', () => {
    const meetups = [{}, {}];
    const expectedAction = {
      type: types.FETCH_MEETUPS_SUCCEEDED,
      meetups
    };
    expect(actions.getAllMeetupsSucceeded(meetups)).toEqual(expectedAction);
  });

  it('it should create an action of type FETCH_MEETUPS_FAILED', () => {
    const error = {};
    const expectedAction = {
      type: types.FETCH_MEETUPS_FAILED,
      error
    };
    expect(actions.getAllMeetupsFailed(error)).toEqual(expectedAction);
  });

  it('it should create an action of type FETCH_SPECIFIC_MEETUP_START', () => {
    const expectedAction = {
      type: types.FETCH_SPECIFIC_MEETUP_START
    };
    expect(actions.getSpecificMeetupStart()).toEqual(expectedAction);
  });

  it('it should create an action of type FETCH_SPECIFIC_MEETUP_SUCCEEDED', () => {
    const meetup = { };
    const expectedAction = {
      type: types.FETCH_SPECIFIC_MEETUP_SUCCEEDED,
      meetup
    };
    expect(actions.getSpecificMeetupSucceeded(meetup)).toEqual(expectedAction);
  });

  it('it should create an action of type FETCH_SPECIFIC_MEETUP_FAILED', () => {
    const expectedAction = {
      type: types.FETCH_SPECIFIC_MEETUP_FAILED
    };
    expect(actions.getSpecificMeetupFailed()).toEqual(expectedAction);
  });

  it('should create an action of type FETCH_SPECIFIC_MEETUP_START and FETCH_SPECIFIC_MEETUP_SUCCEEDED if successful', () => {
    mock.onGet('/meetups/23').reply(200, {
      data: {
        data: []
      }
    });

    const expectedActions = [
      'FETCH_SPECIFIC_MEETUP_START',
      'FETCH_SPECIFIC_MEETUP_SUCCEEDED'
    ];
    const store = mockStore({});
    return store.dispatch(actions.getSpecificMeetup(23))
      .then((response) => {
        const actualActions = store.getActions().map(action => action.type);
        expect(actualActions).toEqual(expectedActions);
      });
  }, 10000);

  it('should create an action of type FETCH_SPECIFIC_MEETUP_START and FETCH_SPECIFIC_MEETUP_FAILED if not successful', () => {
    mock.onGet('/meetups/23').reply(500);

    const expectedActions = [
      'FETCH_SPECIFIC_MEETUP_START',
      'FETCH_SPECIFIC_MEETUP_FAILED'
    ];
    const store = mockStore({});
    return store.dispatch(actions.getSpecificMeetup(23))
      .then((response) => {
        const actualActions = store.getActions().map(action => action.type);
        expect(actualActions).toEqual(expectedActions);
      });
  }, 10000);

  it('should create an action of type FETCH_MEETUPS_START and FETCH_MEETUPS_FAILED if  not successful', () => {
    mock.onGet('/meetups/23').reply(500);

    const expectedActions = [
      'FETCH_MEETUPS_START',
      'FETCH_MEETUPS_FAILED'
    ];
    const store = mockStore({});
    return store.dispatch(actions.getAllMeetups(23))
      .then((response) => {
        const actualActions = store.getActions().map(action => action.type);
        expect(actualActions).toEqual(expectedActions);
      });
  }, 10000);

  it('should create an action of type FETCH_MEETUPS_START and FETCH_MEETUPS_SUCCEEDED if  successful', () => {
    mock.onGet('/meetups/23').reply(200, {
      data: {
        data: []
      }
    });
    const expectedActions = [
      'FETCH_MEETUPS_START',
      'FETCH_MEETUPS_SUCCEEDED'
    ];
    const store = mockStore({});
    store.dispatch(actions.getAllMeetups())
      .then((response) => {
        const actualActions = store.getActions().map(action => action.type);
        expect(actualActions).toEqual(expectedActions);
      });
  }, 10000);
});

describe('meetups reducer', () => {
  const initialState = {
    gettingMeetups: false,
    specificMeetup: {},
    meetups: [],
    error: null,
    loadingSpecificMeetup: false,
    loadSpecificMeetupFailed: false,
  };
  it('should return the initial state', () => {
    expect(AllMeetupReducer(undefined, {})).toEqual({
      gettingMeetups: false,
      specificMeetup: {},
      meetups: [],
      error: null,
      loadingSpecificMeetup: false,
      loadSpecificMeetupFailed: false,
    });
  });

  it('should return state with gettingMeetups set to true and error set to null', () => {
    expect(AllMeetupReducer(initialState, actions.getAllMeetupsStart())).toEqual({
      ...initialState,
      gettingMeetups: true,
      error: null
    });
  });

  it('should return state with gettingMeetups set to false, and meetups array and error set to null', () => {
    const meetup = [{}, {}];
    expect(AllMeetupReducer(initialState, actions.getAllMeetupsSucceeded(meetup))).toEqual({
      ...initialState,
      meetups: [{}, {}],
      gettingMeetups: false,
    });
  });

  it('getting all meetups failed', () => {
    const error = {};
    expect(AllMeetupReducer(initialState, actions.getAllMeetupsFailed(error))).toEqual({
      ...initialState,
      gettingMeetups: false,
      error: {}
    });
  });

  it('start getting specific meetup', () => {
    expect(AllMeetupReducer(initialState, actions.getSpecificMeetupStart())).toEqual({
      ...initialState,
      loadingSpecificMeetup: true
    });
  });

  it('getting specific meetup succeeded', () => {
    const meetup = {};
    expect(AllMeetupReducer(initialState, actions.getSpecificMeetupSucceeded(meetup))).toEqual({
      ...initialState,
      loadingSpecificMeetup: false,
      specificMeetup: {}
    });
  });

  it('getting specific meetup failed', () => {
    expect(AllMeetupReducer(initialState, actions.getSpecificMeetupFailed())).toEqual({
      ...initialState,
      loadingSpecificMeetup: false,
      loadingSpecificMeetupFailed: true
    });
  });
});
