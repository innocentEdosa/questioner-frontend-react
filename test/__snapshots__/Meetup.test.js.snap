// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MeetupJumbotron /> Should match the snapshot 1`] = `
<div
  className="jumbotron jumbotron-fluid specificMeetup"
>
  <div
    className="container"
  >
    <h1
      className="mt-md-n2 specificMeetupMainText"
    />
    <p
      className="lead specificMeetupSubText pt-1 pb-3"
    />
    <div
      className="specificMeetupRsvp"
      style={
        Object {
          "width": "39%",
        }
      }
    >
      <div
        className="specificMeetupRsvp-back"
      >
        <div
          aria-label="Third group"
          className="btn-group"
          role="group"
        >
          <button
            className="btn specificMeetupRsvpBtn"
            onClick={[Function]}
            type="button"
          >
            Yes
          </button>
        </div>
        <div
          aria-label="Third group"
          className="btn-group mx-4"
          role="group"
        >
          <button
            className="btn specificMeetupRsvpBtn"
            onClick={[Function]}
            type="button"
          >
            Maybe
          </button>
        </div>
        <div
          aria-label="Third group"
          className="btn-group"
          role="group"
        >
          <button
            className="btn specificMeetupRsvpBtn"
            onClick={[Function]}
            type="button"
          >
            No
          </button>
        </div>
      </div>
      <div
        className="specificMeetupRsvp-side specificMeetupRsvp-front"
      >
        Add to schedule
      </div>
    </div>
  </div>
</div>
`;

exports[`<ScheduleCard /> Should match the snapshot 1`] = `
<div
  className="specificMeetupRsvp"
  style={Object {}}
>
  <div
    className="specificMeetupRsvp-back"
  >
    <div
      aria-label="Third group"
      className="btn-group"
      role="group"
    >
      <button
        className="btn specificMeetupRsvpBtn"
        onClick={[Function]}
        type="button"
      >
        Yes
      </button>
    </div>
    <div
      aria-label="Third group"
      className="btn-group mx-4"
      role="group"
    >
      <button
        className="btn specificMeetupRsvpBtn"
        onClick={[Function]}
        type="button"
      >
        Maybe
      </button>
    </div>
    <div
      aria-label="Third group"
      className="btn-group"
      role="group"
    >
      <button
        className="btn specificMeetupRsvpBtn"
        onClick={[Function]}
        type="button"
      >
        No
      </button>
    </div>
  </div>
  <div
    className="specificMeetupRsvp-side specificMeetupRsvp-front"
  >
    Add to schedule
  </div>
</div>
`;
